modern/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const traccarPurple = '#333366';
+const traccarGreen = '#4CAF50';
+const traccarWhite = '#FFF';
+
+export default createMuiTheme({
+  palette: {  
+    common: {
+      purple: traccarPurple,
+      green: traccarGreen
+    },
+    primary: {
+      main: traccarPurple
+    },
+    secondary: {
+      main: traccarGreen,
+      contrastText: traccarWhite
+    }
+  },
+  overrides: {
+    MuiFormControl: {
+      root: {
+        height: '42px',
```

Comment:
Do we need to modify the sizes?

Reply:
The input in our design is 42px. So it becomes necessary to modify it here

Reply:
The input in our design is 42px. So it becomes necessary to modify it here

Reply:
In that case we should have a place to store common sizes. I see that there are a bunch of hardcoded numbers.

Reply:
Any recommended location? Maybe `sizes.js`

Reply:
Not sure what the best practice is.

Reply:
It's common for react projects to have a `styles` folder which has `theme.js`, `colors.js` and `dimensions.js`. The names obviously differ on project basis, but the idea is usually present. We can go along with something similar or we can just add a single file to the main directory for the dimensions

Reply:
As long as it's consistent, I think it should be fine.

Reply:
Added theme folder which has index.js, palatte.js, dimensions.js and overrides.js. We can also add typography.js if its required.

Reply:
Great